src/System.Management.Automation/engine/remoting/commands/PSRemotingCmdlet.cs

```diff
@@ -996,6 +982,28 @@ protected void ValidateComputerName(String[] computerNames)
             }
         }
 
+        private static string GetSSHConnectionStringParameter(object param)
+        {
+            var paramValue = param as string;
+            if (string.IsNullOrEmpty(paramValue))
+            {
+                throw new PSArgumentException(RemotingErrorIdStrings.InvalidSSHConnectionParameter);
+            }
+
+            return paramValue;
+        }
+
+        private static int GetSSHConnectionIntParameter(object param)
+        {
+            int? paramValue = param as int?;
+            if (paramValue == null)
+            {
+                throw new PSArgumentException(RemotingErrorIdStrings.InvalidSSHConnectionParameter);
+            }
+
+            return paramValue.Value;
+        }
+
```

Comment:
I am able to build with Core 2.0 and this pattern builds successfully.  So I think we are good to use it.